src/what.md

```diff
@@ -4,8 +4,14 @@ It’s actually kind of difficult to define what an operating system is. There
 are a lot of different kinds of operating systems, and they all do different
 kinds of things.
 
-There are some shared goals, however. Let’s try this out as a working
-definition:
+Some things are commonly bundled with operating systems, but are arguably not
+part of the essence of what makes an OS and OS. For example, many operating
+systems are often marketed as coming equipped with an internet browser or email
```

Comment:
Seems a better solution is to use the term "web browser". I'll change it to that.